### PR TITLE
chore: adding sitemap and robots config

### PR DESCRIPTION
**Pull request by jonataspinto**
## Screenshots / Videos
image
## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [x] Tested on Mobile device

### CODE REVIEW

**Review by ramonxm — APPROVED** [review]
LGTM, massa mano!

**Comment by ramonxm** [comment]
o build falhou @jonataspinto

**Comment by ramonxm** [comment]
Uma observação, não temos step de cache para build nas actions, seria interessante adicionar, não?

**Comment by jonataspinto** [comment]
> Uma observação, não temos step de cache para build nas actions, seria interessante adicionar, não?

Tem referencias para compartilhar? Nunca fiz.

**Comment by jonataspinto** [comment]
> o build falhou @jonataspinto

Vou olhar amanhã, não entendi nada do erro

**Comment by ramonxm** [comment]
> > Uma observação, não temos step de cache para build nas actions, seria interessante adicionar, não?
> 
> Tem referencias para compartilhar? Nunca fiz.

Vi numa aula, mas pesquisei e achei esse material que é basicamente o conteúdo da aula.

https://dev.to/mpocock1/how-to-cache-nodemodules-in-github-actions-with-yarn-24eh

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.82 | ✅ |
| First Contentful Paint | 1.2 s |  |
| Time to Interactive | 8.0 s |  |
| Speed Index | 1.4 s |  |
| Total Blocking Time | 210 ms |  |
| Largest Contentful Paint | 3.5 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.88 | ✅ |
| First Contentful Paint | 2.3 s |  |
| Time to Interactive | 5.8 s |  |
| Speed Index | 3.0 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 2.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "build:exp": "next build && next export",
+    "postbuild": "next-sitemap",
```
Comment (ramonxm) [inline package.json:9]: good.

**Review by ramonxm — DISMISSED** [review]

**Comment by ramonxm** [comment]
lgtm

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.72 | ✅ |
| First Contentful Paint | 1.4 s |  |
| Time to Interactive | 7.7 s |  |
| Speed Index | 2.2 s |  |
| Total Blocking Time | 420 ms |  |
| Largest Contentful Paint | 4.1 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.8 | ✅ |
| First Contentful Paint | 2.6 s |  |
| Time to Interactive | 7.3 s |  |
| Speed Index | 4.0 s |  |
| Total Blocking Time | 200 ms |  |
| Largest Contentful Paint | 3.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/components/Subscribe/index.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const Subscribe = () => {
           color="purple"
           href={config.INSCRICOES_URL}
         >
-          <a target="_blank">{`Faça parte!`}</a>
+          <a target="_blank" rel="noopener">{`Faça parte!`}</a>
```
Comment (ramonxm) [inline src/components/Subscribe/index.tsx:30]: Só  o noopener ou o noreferrer também?

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.77 | ✅ |
| First Contentful Paint | 1.4 s |  |
| Time to Interactive | 7.5 s |  |
| Speed Index | 3.5 s |  |
| Total Blocking Time | 410 ms |  |
| Largest Contentful Paint | 3.3 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.79 | ✅ |
| First Contentful Paint | 2.7 s |  |
| Time to Interactive | 7.4 s |  |
| Speed Index | 4.0 s |  |
| Total Blocking Time | 220 ms |  |
| Largest Contentful Paint | 3.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.71 | ✅ |
| First Contentful Paint | 1.2 s |  |
| Time to Interactive | 7.7 s |  |
| Speed Index | 3.5 s |  |
| Total Blocking Time | 330 ms |  |
| Largest Contentful Paint | 4.5 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.89 | ✅ |
| First Contentful Paint | 2.4 s |  |
| Time to Interactive | 5.4 s |  |
| Speed Index | 3.1 s |  |
| Total Blocking Time | 110 ms |  |
| Largest Contentful Paint | 2.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.73 | ✅ |
| First Contentful Paint | 1.3 s |  |
| Time to Interactive | 7.8 s |  |
| Speed Index | 2.0 s |  |
| Total Blocking Time | 280 ms |  |
| Largest Contentful Paint | 4.6 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.8 | ✅ |
| First Contentful Paint | 2.7 s |  |
| Time to Interactive | 7.1 s |  |
| Speed Index | 4.1 s |  |
| Total Blocking Time | 140 ms |  |
| Largest Contentful Paint | 3.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

File: src/components/Subscribe/index.tsx (hunk 2 above)
Comment (jonataspinto) [reply src/components/Subscribe/index.tsx]: Valeu! adicionei

**Pull request merged**